c/column.h

```diff
@@ -118,14 +118,17 @@ class Column
   size_t nrows;
 
 public:
+  static constexpr size_t MAX_STRING_SIZE = 0x7FFFFFFF;
+  static constexpr size_t MAX_STR32_BUFFER_SIZE = 0x7FFFFFFF;
+  static constexpr size_t MAX_STR32_NROWS = 0x7FFFFFFF;
```

Comment:
If it is `size_t`, why we only do `0x7FFFFFFF`? Do we have `int32_t` coming into effect somewhere?

Reply:
This is the maximum size for STR32 column. If actual size is bigger, it has to become STR64